corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -486,6 +486,7 @@
         <![endif]-->
         
         {% include 'style/includes/keyboard_shortcuts.html' %}
+        {% block keyboard_shortcuts %}{% endblock keyboard_shortcuts %}
```

Comment:
looks like this block is already included with https://github.com/dimagi/commcare-hq/blob/app-manager-keyboard-shortcuts/corehq/apps/style/templates/style/includes/keyboard_shortcuts.html#L225 but is wrapped in an if.  i don't have too much context here to say which one should be removed but i think django will raise an exception if it's declared twice